### ISSUE TITLE
Getting loss as nan in train_triplets_semi_hard.py

### ISSUE THREAD

**Issue by yksolanki9** [AKASH2907/deepfakes_video_classification#9]
On training the network, I am getting loss as nan and a valid value for validation loss which decreases with epochs. Any idea as to why I am getting loss as nan and how to avoid it?

**Comment by AKASH2907** [comment]
Try increasing your batch size. What's your current batch size?

**Comment by yksolanki9** [comment]
I increased it to 64 and 128. Still getting nan loss.

**Comment by AKASH2907** [comment]
Try 256 and 512 too. It happened with me also but when I increased the batch size nan loss disappeared.

**Comment by yksolanki9** [comment]
That worked. Thanks!
